my-dsl.md

```diff
@@ -1,38 +1,68 @@
-# Language
+# Body Markup Language (BML)
+
+## Language
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+Behavior Markup Language (BML) 
+http://www.mindmakers.org/projects/bml-1-0/wiki#BML-10-Standard
 
-# Domain
+## Domain
 _Describe the language's domain in five words._
+Controlling embodied conversational agents’ behaviors
 
-
-# Computational model
+## Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
+BML code specifies the behaviors to be performed by an embodied conversation 
+agent - these can include nonverbal facial and body movements as well as 
+verbal behaviors. The code is executed through a module called a BML Realizer, 
+which may be run as a stand-alone application, embedded as a library, or 
+embedded within a game engine. When the BML Realizer is run, an embodied 
+conversation agent that is integrated with the BML Realizer performs the 
+behaviors. 
 
-
-# DSL-ness
+## DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
```

Comment:
I wonder whether it's possible to use BML for more sophisticated applications on its own, without the help of other languages (if possible, this may mean really inelegant code)?